components/tools/OmeroPy/src/omero/util/script_utils.py

```diff
@@ -30,8 +30,11 @@
 from numpy import add, array, asarray, fromstring, reshape, zeros
 from os.path import exists
 
+
```

Comment:
These *double*-blanks are interesting, are they now a standard way we separate our-code imports?